tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -449,126 +436,85 @@ async def test_inner_linear_normalization(
     mock.normalize.assert_not_called()
 
 
-@pytest.mark.parametrize(
-    "classifier_params, output_length",
-    [({LOSS_TYPE: "margin", RANDOM_SEED: 42, EPOCHS: 1}, LABEL_RANKING_LENGTH)],
-)
 async def test_margin_loss_is_not_normalized(
-    monkeypatch, component_builder, tmpdir, classifier_params, output_length
+    monkeypatch: MonkeyPatch, create_train_load_and_process_diet: Callable[..., Message]
 ):
-    pipeline = as_pipeline(
-        "WhitespaceTokenizer", "CountVectorsFeaturizer", "DIETClassifier"
-    )
-    assert pipeline[2]["name"] == "DIETClassifier"
-    pipeline[2].update(classifier_params)
-
     mock = Mock()
     monkeypatch.setattr(train_utils, "normalize", mock.normalize)
 
-    _config = RasaNLUModelConfig({"pipeline": pipeline})
-    (trained_model, _, persisted_path) = await rasa.nlu.train.train(
-        _config,
-        path=str(tmpdir),
-        data="data/test/many_intents.yml",
-        component_builder=component_builder,
+    parsed_message = create_train_load_and_process_diet(
+        {LOSS_TYPE: "margin", RANDOM_SEED: 42, EPOCHS: 1},
+        training_data="data/test/many_intents.yml",
     )
-    loaded = Interpreter.load(persisted_path, component_builder)
-
-    parse_data = loaded.parse("hello")
+    parse_data = parsed_message.data
     intent_ranking = parse_data.get("intent_ranking")
 
     # check that the output was not normalized
     mock.normalize.assert_not_called()
 
     # check that the output was correctly truncated
-    assert len(intent_ranking) == output_length
+    assert len(intent_ranking) == LABEL_RANKING_LENGTH
```

Comment:
was this maybe a parameter because `LABEL_RANKING_LENGTH` is just a default?

Reply:
I think these ended up only having one item in paramaterize after recent deprecation removals - so I just removed the pointless `paramaterize`.